myTrip/auth_reg/backends.py

```diff
@@ -0,0 +1,19 @@
+"""Contains authentication backends."""
+
+from .models import CustomUser
+
+class CustomUserAuth(object):
+
```

Comment:
Nothing to add here since the docstring at the very top of this file says what it does.